### PR TITLE
cyanrip: 0.9.0 -> 0.9.2

### PR DESCRIPTION
**Pull request by vs49688**
## Description of changes

Updating cyanrip to 0.9.2.

Also convert to `pkgs/by-name` and reformat.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by vs49688** [comment]
@ofborg build cyanrip

**Comment by vs49688** [comment]
Result of `nixpkgs-review pr 268459` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cyanrip</li>
  </ul>
</details>

File: pkgs/by-name/cy/cyanrip/package.nix [hunk 1]
```diff
@@ -10,19 +10,30 @@
...
-  version = "0.9.0";
+  version = "0.9.2";
 
   src = fetchFromGitHub {
     owner = "cyanreg";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-gH/rWTRYX10Q2Y9oSaMu0bOy3SMbcSNmH3dkXHFAw90";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
```
Comment (SuperSandro2000) [inline pkgs/by-name/cy/cyanrip/package.nix:20]:
```suggestion
    repo = pname;
    rev = "v${finalAttrs.version}";
```
appending pname surely should not change the download repo

also changing version here will have no effect until the hash is also changed

Comment (vs49688) [reply pkgs/by-name/cy/cyanrip/package.nix:20]: > appending pname surely should not change the download repo

Won't that require me to add `rec` back? I though the point of `finalAttrs` was to remove uses of `rec`. I can just hardcode it, I guess.

> also changing version here will have no effect until the hash is also changed

I did: `hash = "sha256-sWgHf8S4GZDAIvMUf5KvGy2y0JcUbRS53IjArdgokqc=";`

**Review by pbsds — APPROVED** [review]
Result of `nixpkgs-review pr 268459` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cyanrip</li>
  </ul>
</details>

Runs fine, LGTM

**Pull request merged**